000-vip-init.php

```diff
@@ -287,5 +289,13 @@
 	add_filter( 'wp_sitemaps_enabled', '__return_false' );
 }
 
+require_once __DIR__ . '/001-core/constants.php'; // Define the DB constants
+
+// Roll out to non-prods only for now
+if ( defined( 'VIP_GO_APP_ENVIRONMENT' ) && 'production' !== constant( 'VIP_GO_APP_ENVIRONMENT' ) ) {
+	global $wpdb;
+	define_db_constants( $wpdb );
```

Comment:
Any concerns with setting `global` in the file execution like this? Would it be cleaner / safer to do something like this instead:

```suggestion
	define_db_constants( $GLOBALS['wpdb'] );
```

Reply:
++ to what Mo said.